src/turbolinks/history.coffee

```diff
@@ -20,16 +23,27 @@ class Turbolinks.History
 
   push: (location, restorationIdentifier) ->
     location = Turbolinks.Location.wrap(location)
+    locationStack.push(location)
     @update("push", location, restorationIdentifier)
 
   replace: (location, restorationIdentifier) ->
     location = Turbolinks.Location.wrap(location)
     @update("replace", location, restorationIdentifier)
 
+  back: (location, restorationIdentifier) ->
```

Comment:
I feel like `refresh` sounds synonymous to `reload` as in "refresh the page", which would suggest history doesn't move. In this case, history is indeed going "back". Other synonyms of `go_back`, `retire`, `retreat`, none of those sound good! I think `return` makes the most sense, but it's also a bit confusing. If you were instructing someone to go to the previous page, how would you say it? "Hit back" or "Go back". I guess you could say "Return to the previous page" but you could just as easily say "Return to the home screen". "Return" by itself doesn't accurately describe it as well as "go back". Hmmm!

There's my two cents. You can pick whatever name you think is best and I'll update it :)